packages/desktop-gui/src/specs/specs-list.jsx

```diff
@@ -156,6 +165,28 @@ class SpecsList extends Component {
     return spec.hasChildren ? this._folderContent(spec, nestingLevel) : this._specContent(spec, nestingLevel)
   }
 
+  _setFocus = () => {
+    this.setState({
+      isFocused: !this.state.isFocused,
+    })
+  }
+
+  _setBlur = () => {
+    this.setState({
+      isFocused: !this.state.isFocused,
+    })
+  }
+
+  _isMac () {
+    return (window.clientInformation['platform'] === 'MacIntel')
```

Comment:
@jennifer-shehane in case you're wondering about this. It's a[ workaround of a known issue](https://github.com/dopecodez/pingman/issues/10#issuecomment-678344075).